.travis.yml

```diff
@@ -1,10 +1,36 @@
-language: java
-before_install:
-  - bash dev/travis-before_install.sh
+language: generic
+
+services:
+  - docker
 
 env:
-  - HADOOP_PROFILE=default TEST_CODECS=uncompressed,brotli
-  - HADOOP_PROFILE=default TEST_CODECS=gzip,snappy
+  global:
+    - JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
```

Comment:
I would move this as an `ENV` step in the docker image. It is part of the docker container, instead of the Travis build.

Reply:
No, part of the work here is to use the same image to test with Java 11 in the future if needed, and the thrift image includes Java 11. I know it looks weird but this will allow us to build two separate images.

Reply:
Thanks. Just to clarify, the `JAVA_HOME` is also set in the base image, right? https://github.com/docker-library/openjdk/blob/master/8/jdk/Dockerfile#L20

For JDK11 we're still waiting on some blockers: https://github.com/apache/parquet-mr/pull/596 Mostly waiting for a new release of Parquet Format https://github.com/apache/parquet-format/pull/127. This enables us to update Scrooge, which is required to update to Java 11.

Reply:
Actually the parent image of this Parquet one is not `openjdk`, it is `thrift`, and that's the reason why we had to define it manually, and why I had to install manually the JDK in the scripts too.

Reply:
Thanks for the links on Java 11 I wanted to be catch the progress of this :+1:

Reply:
Minor question what happened to the 1.11.0 release, i saw we are already in 1.12.0-SNAPSHOT but have not seen a release any part.

Reply:
My mistake, I was under the impression that it was being built from `openjdk`, but `thrift` makes more sense. Thanks for the clarification! 👍 

The 0.11 release is still in progress, but has stalled because of unsigned keys:
https://lists.apache.org/thread.html/df101f14f5b5a2489443e23be32c7342e2b8a6c8eb95f2e6953fd5b4@%3Cdev.parquet.apache.org%3E